doc/api/readline.md

```diff
@@ -25,121 +27,61 @@ rl.question('What do you think of Node.js? ', (answer) => {
 });
 ```
 
-## Class: Interface
-
-The class that represents a readline interface with an input and output
-stream.
-
-### rl.close()
-
-Closes the `Interface` instance, relinquishing control on the `input` and
-`output` streams. The `'close'` event will also be emitted.
-
-### rl.pause()
-
-Pauses the readline `input` stream, allowing it to be resumed later if needed.
-
-Note that this doesn't immediately pause the stream of events. Several events may
-be emitted after calling `pause`, including `line`.
-
-### rl.prompt([preserveCursor])
-
-Readies readline for input from the user, putting the current `setPrompt`
-options on a new line, giving the user a new spot to write. Set `preserveCursor`
-to `true` to prevent the cursor placement being reset to `0`.
-
-This will also resume the `input` stream used with `createInterface` if it has
-been paused.
-
-If `output` is set to `null` or `undefined` when calling `createInterface`, the
-prompt is not written.
-
-### rl.question(query, callback)
-
-Prepends the prompt with `query` and invokes `callback` with the user's
-response. Displays the query to the user, and then invokes `callback`
-with the user's response after it has been typed.
-
-This will also resume the `input` stream used with `createInterface` if
-it has been paused.
-
-If `output` is set to `null` or `undefined` when calling `createInterface`,
-nothing is displayed.
+*Note* Once this code is invoked, the Node.js application will not
+terminate until the `readline` interface is closed.
 
-Example usage:
-
-```js
-rl.question('What is your favorite food?', (answer) => {
-  console.log(`Oh, so your favorite food is ${answer}`);
-});
-```
-
-### rl.resume()
-
-Resumes the readline `input` stream.
-
-### rl.setPrompt(prompt)
-
-Sets the prompt, for example when you run `node` on the command line, you see
-`> `, which is Node.js's prompt.
-
-### rl.write(data[, key])
-
-Writes `data` to `output` stream, unless `output` is set to `null` or
-`undefined` when calling `createInterface`. `key` is an object literal to
-represent a key sequence; available if the terminal is a TTY.
-
-This will also resume the `input` stream if it has been paused.
-
-Example:
-
-```js
-rl.write('Delete me!');
-// Simulate ctrl+u to delete the line written previously
-rl.write(null, {ctrl: true, name: 'u'});
-```
+## Class: Interface
 
-## Events
+The `readline.Interface` class is constructed using the
```

Comment:
nit: more of a technicality, but ``Instances of the `readline.Interface` class are constructed``?